rc/filetype/nim.kak

```diff
@@ -66,7 +76,7 @@ evaluate-commands %sh{
     types="int int8 int16 int32 int64 uint uint8 uint16 uint32 uint64 float
     float32 float64 bool char object seq array cstring string tuple varargs
     typedesc pointer byte set typed untyped void auto"
-    values="false true"
+    values="false true on off"
```

Comment:
only place where on/off are used are pragmas for compiler options.

Reply:
No, they actually are aliases for true and false. I use them all the time. Try this on play.nim-lang.org ;-)
```nim
let is_on_an_alias_for_true = on

if is_on_an_alias_for_true != off:
  echo "YES!!!"
```

Reply:
thats cool that they are not even mentioned in manual or grammar. i guess that was a hack made specifically for pragmas.

Reply:
Yeah, Nim's manual is perennially lagging behind the actual specification.